site/src/modules/provisioners/ProvisionerGroup.tsx

```diff
@@ -166,67 +166,45 @@ export const ProvisionerGroup: FC<ProvisionerGroupProps> = ({
 								borderRadius: 8,
 								border: `1px solid ${theme.palette.divider}`,
 								fontSize: 14,
-								padding: "12px 18px",
-								width: 310,
+								padding: "14px 18px",
+								width: 375,
```

Comment:
Could the `width` be swapped out for `flex-basis`?

Reply:
I kinda don't want them to grow tho, and disrupt the "grid"